src/update.js

```diff
@@ -23,6 +23,10 @@ function err(operation, expectedTarget, path) {
   );
 }
 
+const tr = true;
+const arrayMethods = {push: tr, unshift: tr, concat: tr, splice: tr, pop: tr, shift: tr};
```

Comment:
Will be nice to use Set.
@Yomguithereal  what do you think about drop IE9 and IE10?